portal/assets/js/global.js

```diff
@@ -68,4 +68,21 @@ document.addEventListener('DOMContentLoaded', function() {
     addMaildrop.parentNode.insertBefore(inputDropClone, addMaildrop);
   });
 
+  // FIXME - I don't understand what this do ...
+  // This looks kinda hackish :|
+  if(window.location != window.parent.location) {
+    // Set class to body to show we're in overlay
+    document.body.classList.add('overlay');
+      let userContainer = document.querySelector('a.user-container');
+      userContainer.classList.replace('user-container-info', 'user-container-edit');
+      userContainer.setAttribute('href', userContainer
```

Comment:
This changes the `href` from `[...]/edit.html` to `[...]/` (which becomes `[...]/info.html` eg. the main page)